src/common/options/rgw.yaml.in

```diff
@@ -2123,6 +2123,18 @@ options:
   services:
   - rgw
   with_legacy: true
+- name: rgw_data_log_recovery_interval
+  type: int
+  level: advanced
+  desc: How often to run the datalog recovery process
+  long_desc: The data log keeps a collection of per-bucketshard
+    semaphores for crash recovery. Recovery runs at startup. If
+    desires, it can run periodically.
+  fmt_desc: The recovery interval in seconds, 0 to disable except at startup.
+  default: 0
+  services:
+  - rgw
+  with_legacy: true
```

Comment:
i don't think we should be using 'legacy' options by default. in this case, i don't think we care about the overhead of a mutex in `renew_run()`

Reply:
Just removed the periodic rerun. I'll add a radosgw-admin command to run it in another PR like I mentioned on slack, but I don't want to hold this up further.